app.rb

```diff
@@ -0,0 +1,58 @@
+require 'pg'
+require 'sinatra'
+require './lib/user'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+
+  get '/' do
+    @peeps = Peep.all
+
+    return erb :index if session[:user_id].nil?
+
+    user = User.find(id: session[:user_id])
+    @name = user.name
+    erb :logged_in
+  end
+
+  post '/post_peep' do
```

Comment:
Remember to use 'RESTful' routes. e.g. post '/peeps' for creating a new peep, get 'peeps/:id' for viewing a single peep